be/src/vec/exec/format/table/iceberg_reader.cpp

```diff
@@ -122,6 +144,16 @@ Status IcebergTableReader::init_row_filters() {
         auto files = table_desc.delete_files;
         if (delete_file_type == POSITON_DELELE) {
             // position delete
+            auto row_desc = RowDescriptor(_state->desc_tbl(),
+                                          std::vector<TupleId>({table_desc.delete_table_tuple_id}),
+                                          std::vector<bool>({false}));
+            if (_data_path_conjunct_ctx == nullptr) {
```

Comment:
No need to check `if (_data_path_conjunct_ctx == nullptr)`, it must be null here.